tools/topology/topology2/sof-ace-tplg/tplg-targets.cmake

```diff
@@ -53,9 +53,9 @@ DEEPBUFFER_D0I3_COMPATIBLE=true"
 "cavs-rt5682\;sof-mtl-max98357a-rt5682-ssp2-ssp0-2ch-pdm1\;PLATFORM=mtl,NUM_DMICS=2,\
 PDM0_MIC_A_ENABLE=0,PDM0_MIC_B_ENABLE=0,PDM1_MIC_A_ENABLE=1,PDM1_MIC_B_ENABLE=1,\
 DMIC0_PCM_ID=99,PREPROCESS_PLUGINS=nhlt,NHLT_BIN=nhlt-sof-mtl-max98357a-rt5682-2ch.bin,\
-HEADSET_SSP_DAI_INDEX=2,SPEAKER_SSP_DAI_INDEX=0,HEADSET_CODEC_NAME=SSP2-Codec,SPEAKER_CODEC_NAME=SSP0-Codec,\
-BT_NAME=SSP1-BT,BT_INDEX=1,BT_ID=8,BT_PCM_NAME=Bluetooth,INCLUDE_ECHO_REF=true,USE_CHAIN_DMA=true,\
-DEEPBUFFER_FW_DMA_MS=10,DEEPBUFFER_D0I3_COMPATIBLE=true"
+NUM_HDMI=3,HEADSET_SSP_DAI_INDEX=2,SPEAKER_SSP_DAI_INDEX=0,SPK_ID=6,HEADSET_CODEC_NAME=SSP2-Codec,\
```

Comment:
it's NUM_HDMIS=3 and this should be set for all devices.

@ranj063 @bardliao can we make this platform-dependent so that we don't have this sort of issues?

Reply:
> it's NUM_HDMIS=3 and this should be set for all devices.
> 
> @ranj063 @bardliao can we make this platform-dependent so that we don't have this sort of issues?

Yes, we can define NUM_HDMI in platform/intel/mtl.conf

Reply:
@bardliao @plbossart - If this is included in the mtl.conf, other topologies would affect as below information still exists for one configuration
<img width="444" alt="image" src="https://github.com/thesofproject/sof/assets/62392729/fb846b98-cabc-4354-b0f6-b698d70a7edf">

https://github.com/thesofproject/linux/blob/topic/sof-dev/sound/soc/intel/boards/sof_rt5682.c#L188

Should we need to update with NUM_DMICS=4 for these topologies?

Reply:
@udaymb if there's 1 topology with NUM_HDMIS=4, then thats the one that needs to override it at build time. Please make 3 the default and set it to 4 where needed

Reply:
Updated the commit